atom/browser/api/atom_api_web_contents.cc

```diff
@@ -949,7 +951,26 @@ bool WebContents::HandleContextMenu(const content::ContextMenuParams& params) {
     Emit("pepper-context-menu", std::make_pair(params, web_contents()));
     web_contents()->NotifyContextMenuClosed(params.custom_context);
   } else {
-    Emit("context-menu", std::make_pair(params, web_contents()));
+    // For forgetting custom dictionary option
+    content::ContextMenuParams new_params(params);
+    if (new_params.misspelled_word.empty() &&
```

Comment:
actually we should also check `is_editable` and `spellcheck_enabled` because spellcheck only applies to editable fields. Just want to narrow this check to run as infrequently as possible